### PR TITLE
feat: Adding contract addresses

### LINKED ISSUE #140: [Feature]: System Contracts deployment addresses
### Description

Right now, it's really difficult to find the system contracts deployment addresses, please add them to the documentation for testing. 

### Rationale

When I test my smart contracts, I want to make sure I'm interacting with the correct addresses. 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by mm-zk**
# Description

Adding contract addresses for mainnet and testnet.

Fixes #140

### CODE REVIEW

File: content/00.build/95.resources/graph.png
Comment (MexicanAce) [inline content/00.build/95.resources/graph.png:1]: We should reference the existing image as this one still uses `Hyperchain`, which is term we need to stop referencing

File: content/00.build/95.resources/80.addresses.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+| Cronos testnet  | 282 | Chronos testnet | 0xeBf961DC3904725ec916678D875B9625d5F7C29f | |
+
+
+
+# How to get this data.
+
+Start from Bridgehub.
+To get the STM, call stateTransitionManager (and pass chain id)
+
+in stateTransitionManager, you can query the StateTransition (a.k.a Hyperchain) contract.
```
Comment (MexicanAce) [inline content/00.build/95.resources/80.addresses.md:59]:
```suggestion
in stateTransitionManager, you can query the StateTransition (a.k.a ZK Chain) contract.
```

File: content/00.build/95.resources/80.addresses.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+| ?? | 444 | ?? | 0x2F45Fc02ec23e6D335fdE7B5a5083F053C8C2aD3 | |
+| ?? | 531050104 | ?? | 0xDd02368f378EeD3779B358D2020a77e4D022236D | |
+| GRVT | 326 | Gravity Sepolia testnet | 0x0e4Bd2E8D953a0234Fb1122FFB848B49522308eC | |
+| ?? | 327 | ?? | 0xB484A3F98bCbAAf50d66292463203c0247B4f3F6 | |
+
+
+## STM (custom - Cronos)
+STM: 0x762b5F15CAd9880ace81776f9046d6a52DD67a9b
+
+Hyperchains:
```
Comment (MexicanAce) [inline content/00.build/95.resources/80.addresses.md:47]:
```suggestion
ZK Chains:
```

File: content/00.build/95.resources/80.addresses.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+# Testnet
+
+
+Bridgehub:  https://sepolia.etherscan.io/address/0x35A54c8C757806eB6820629bc82d90E056394C92
+SharedBridge: https://sepolia.etherscan.io/address/0x3E8b2fe58675126ed30d0d12dea2A9bda72D18Ae
+
+## STM (official)
+STM: 0x4e39E90746A9ee410A8Ce173C7B96D3AfEd444a5
+
+Hyperchains:
```
Comment (MexicanAce) [inline content/00.build/95.resources/80.addresses.md:32]:
```suggestion
ZK Chains:
```

File: content/00.build/95.resources/graph.png
Comment (itsacoyote) [reply content/00.build/95.resources/graph.png:1]: Where is the up to date image?

File: content/00.build/65.developer-reference/12.addresses/80.zk-chain-addresses.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+## Testnet
+
+Bridgehub:  https://sepolia.etherscan.io/address/0x35A54c8C757806eB6820629bc82d90E056394C92
+SharedBridge: https://sepolia.etherscan.io/address/0x3E8b2fe58675126ed30d0d12dea2A9bda72D18Ae
+
+### STM (official)
+STM: 0x4e39E90746A9ee410A8Ce173C7B96D3AfEd444a5
+
+Hyperchains:
```
Comment (jennyg0) [inline content/00.build/65.developer-reference/12.addresses/80.zk-chain-addresses.md:44]: should it be hyperchain here

File: content/00.build/65.developer-reference/12.addresses/80.zk-chain-addresses.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+Learn more about [Shared Bridges](/zk-stack/components/shared-bridges) in the ZK Stack section.
+To understand the concept of ZK Chains, learn more on [ZK Chains](/zk-stack/concepts/zk-chains).
+
+## Getting the State Transition Manager for a ZK Chain
+
+The Bridgehub smart contract has a read method `stateTransitionManager` that expects a chain ID as an argument.
+The return value is the address of the StateTransition (a.k.a ZK Chain) contract address.
+
+![graph.png](/images/developer-reference/l1-smart-contracts/Hyperchain-scheme.png)
```
Comment (jennyg0) [inline content/00.build/65.developer-reference/12.addresses/80.zk-chain-addresses.md:18]: and here

**Review by jennyg0 — APPROVED** [review]

**Pull request merged**